cmake/std/atdm/utils/get_known_system_name.sh

```diff
@@ -69,12 +70,10 @@ elif [[ $ATDM_CONFIG_REAL_HOSTNAME == "ghost"* ]] \
 # tlcc2 systems
 elif [[ $SNLSYSTEM == "tlcc2"* ]] ; then
   ATDM_SYSTEM_NAME=tlcc2
-  if [[ $ATDM_HOSTNAME == "chama"* ]] ; then
-    ATDM_HOSTNAME=chama
-  elif [[ $ATDM_HOSTNAME == "skybridge"* ]] ; then
-    ATDM_HOSTNAME=skybridge
-  else
+  if [[ $SNLCLUSTER == "" ]] ; then
     ATDM_HOSTNAME=$ATDM_CONFIG_REAL_HOSTNAME
+  else
+    ATDM_HOSTNAME=$SNLCLUSTER
   fi
```

Comment:
Okay, now this is completely general.  This will now work for any 'tlcc2' cluster, right?

Reply:
yes it is my understanding that all tlcc2 use SNLSYSTEM and SNLCLUSTER so this should work on any of them